### ISSUE TITLE
Origin.exe not responding on login window when DXVK enabled

### ISSUE THREAD

**Issue by pogof** [lutris/docs#85]
As the title says. Installed Origin just a few minutes ago on Pop_OS 21.10 (nvidia). However after launching the login prompt would just be "transparent" as a took on whatever was under it and in few seconds origin.exe is not responding would appear. After disabling DXVK in the config tab it works as intended.

Might be a good idea to add it to the wiki article.

**Comment by tannisroot** [comment]
I cannot reproduce this issue

**Comment by tannisroot** [comment]
I will attempt to resolve this on DXVK's side

**Comment by pogof** [comment]
Issue went away after i logged in. Then when I re-enabled DXVK it works again and as stated elsewhere its needed for actually running games. (Got error message about missing dll when I tried to run The Sims 2 without it enabled)

The thing that i forgot to mention is that when I went to download all the needed dependencies for wine and specially for origin apt told me that I already have everything as I was messing with wine previously. Which I assume means that I had everything setup correctly.

Freak accident? Or maybe it just plagues the login prompt?
